.github/workflows/hardhat-core-ci.yml

```diff
@@ -267,26 +266,41 @@ jobs:
 
       - uses: Swatinem/rust-cache@v2
 
-      - name: Doctests
+      - name: Run cargo test
         uses: actions-rs/cargo@v1
         env:
           RUSTFLAGS: ${{ matrix.RUSTFLAGS }}
         with:
           command: test
-          args: --doc --workspace --all-features
+          args: --workspace --all-targets --features tracing,bench-once
```

Comment:
This needs the ALCHEMY_URL env var in order to run the remote tests: https://github.com/NomicFoundation/hardhat/actions/runs/4939180231/jobs/8829714657#step:5:208

Reply:
Good catch. I added it to the doc tests by accident instead